pydax/_schema.py

```diff
@@ -108,6 +127,11 @@ class SchemaManager():
 
     def __init__(self, **kwargs: Schema) -> None:
         """Constructor method
+
+        Example:
+
+        >>> dataset_schema = Schema('./tests/schemata/datasets.yaml')
+        >>> schema_manager = SchemaManager(datasets=dataset_schema)
```

Comment:
To me this example should tell something. I don't think a user who reads this document would really gain anything. Perhaps print `schema_manager.schemata`?